stories/stockfish/engine.ts

```diff
@@ -0,0 +1,42 @@
+const stockfish = new Worker("./stockfish.js");
```

Comment:
This may need to be added as a comment in the storybook component above so users can see it in the story

Reply:
You mean entire file?

Reply:
maybe a better solution would be to add the URL to this file in github as the comment instead?